### ISSUE TITLE
Androidx86 Intel HD530 (Core i5 6500T) - H264 is not available.

### ISSUE THREAD

**Issue by ruthan** [moonlight-stream/moonlight-android#570]
Hello,
im trying this app on AndroidX86, OpengES 3.2 is there, games are working, but Moonlight streaming doesnt with error, that H264 is not available.

  Could you fix it?

**Comment by cgutman** [comment]
Android x86 probably doesn't implement a hardware H.264 decoder. Can install the AIDA64 app and screenshot the Codecs tab?

**Comment by yutouyes** [comment]
Same issue. AIDA64 screenshots below
![screenshot_20180726-200429](https://user-images.githubusercontent.com/17039457/43261466-54bc94c0-910f-11e8-8382-e6a94b31c678.jpg)
![screenshot_20180726-200437](https://user-images.githubusercontent.com/17039457/43261469-56bc0ad0-910f-11e8-9514-aa248aa956d4.jpg)
![screenshot_20180726-200442](https://user-images.githubusercontent.com/17039457/43261474-595646f2-910f-11e8-9b74-7acf3a7ab03b.jpg)
![screenshot_20180726-200449](https://user-images.githubusercontent.com/17039457/43261477-5b3363c4-910f-11e8-8a75-e963536ac48b.jpg)
![screenshot_20180726-200452](https://user-images.githubusercontent.com/17039457/43261479-5d71d38c-910f-11e8-8eff-0f406ed5e52d.jpg)
Seems both H.264 decoder&encoder detected. But moonlight said "This device or ROM doesn't support hardware accelerated H.264 playback"

**Comment by cgutman** [comment]
Yeah, I believe ffmpeg decoders are not using hardware acceleration, so Moonlight avoids them.

**Comment by foxtrotdragon** [comment]
I made a pull request that should fix the issue by allowing omx.google to be used, from my tests the it seems to be 1 second behind in latency but the laptop im using is quite well terrible to be blunt

but I did get it to run on Phoenix OS, Bliss OS, and Android TV x86 from TOSAnthony over on kda with similar preformance except for bliss which barely ran at all, but again the laptop im using is not the best by any regards
